sunpy/util/exceptions.py

```diff
@@ -4,10 +4,14 @@
 sunpy. Exceptions that are specific to a given subpackage should *not*
 be here, but rather in the particular subpackage.
 """
+from astropy.utils.exceptions import AstropyWarning
```

Comment:
Do we need to use `AstropyWarning`?

Reply:
Yes we do because we are using the AstroPy logging system and it is designed to specifically ignore the AstropyWarnings. We'd have to make larger changes if we wanted to do something different.